contracts/ConnextModule.sol

```diff
@@ -63,17 +63,30 @@ contract ConnextModule is Module {
         emit ModuleSetUp(owner(), avatar, target, originAddress, origin, executor);
     }
 
-    modifier onlyExecutor(bytes memory _message) {
+    modifier onlyExecutor(address _originSender, uint32 _origin) {
         if (msg.sender != executor) revert ExecutorOnly();
-        if (LibCrossDomainProperty.originSender(_message) != originAddress) revert OriginAddressOnly();
-        if (LibCrossDomainProperty.origin(_message) != origin) revert OriginOnly();
+        if (_originSender != originAddress) revert OriginAddressOnly();
+        if (_origin != origin) revert OriginOnly();
```

Comment:
this is more an implementation question, but if this is going to process multiple transactions from multiple origins, would we want the addresses allowed to be a mapping keyed on domain? I'm unsure if this is how it will be used

Reply:
based on feedback from Auryn, it will be multi-destination instead so we can keep this as a single value.